cli/internal/mock/plugin.go

```diff
@@ -50,8 +75,8 @@ func (p *mockPlugin) GenerateEnvelope(ctx context.Context, req *plugin.GenerateE
 }
 
 func (p *mockPlugin) VerifySignature(ctx context.Context, req *plugin.VerifySignatureRequest) (*plugin.VerifySignatureResponse, error) {
-	if p.fail {
-		return nil, fmt.Errorf("expected error")
+	if p.fail || !p.envGenerator {
```

Comment:
why do we need to check `!p.envGenerator` here?

Reply:
Well we dont need this condition so removed.